src/v2/Components/Artwork/Details.tsx

```diff
@@ -49,54 +59,79 @@ export class Details extends React.Component<Props, null> {
         .slice(1)
       return (
         <TruncatedLine>
-          <strong>{artistLine}</strong>
+          <Sans size="3t" color="black100" weight="medium">
+            {artistLine}
+          </Sans>
         </TruncatedLine>
       )
     }
   }
 
   titleLine() {
     const { includeLinks } = this.props
+    const title = this.props?.artwork?.title
+    const date = this.props?.artwork?.date
+    const href = this.props?.artwork?.href
+
     const artworkText = (
-      <>
-        <em>{this.props.artwork.title}</em>
-        {this.props.artwork.date && `, ${this.props.artwork.date}`}
-      </>
-    )
-    const artworkTextWithLink = includeLinks ? (
-      <TextLink href={this.props.artwork.href}>{artworkText}</TextLink>
-    ) : (
-      artworkText
+      <TruncatedLine>
+        <Sans size="3t" color="black60">
+          {date ? title + ", " + date : title}
```

Comment:
I think this code is predates the `imageTitle` field in MP and instead of concatenating date and title we can just use this here.